sources/packages/web/src/views/aest/institution/Profile.vue

```diff
@@ -57,6 +57,11 @@
               propertyTitle="Regulating body"
               :propertyValue="institutionProfileDetail.regulatingBody"
             />
+            <title-value
+              v-if="institutionProfileDetail.regulatingBody === `other`"
```

Comment:
`v-if="institutionProfileDetail.regulatingBody === 'other'"`

Reply:
If I will not use the single quotes, other won't be recognized in the template as a string with which `institutionProfileDetail.regulatingBody` is compared.

Reply:
no... my comment was to use single quotes instead of ```

Reply:
My bad. I will replace the backticks with the single quote.